docker/README.md

```diff
@@ -0,0 +1,46 @@
+README
+======
+
+This folder contains a Dockerfile and some utilities that simplify the creation of a Docker container for Personfinder development and testing.
+
+Steps to create the container
+-----------------------------
+
+1. Setup the environment for Docker (see Docker's webpage)
+2. From `personfinder/docker` folder, run
+
+    $ docker build -t <image-name> .
+
+    e.g.
+
+    $ docker build -t personfinder-image
+
+    This will take a while and will create the docker image.
+
+3. Create the container running
+
+    $ ./run-container.sh <image-name> [<container-name>]
+
+    e.g.
+
+    $ ./run-container.sh personfinder-image personfinder-container  # Container's name is optional
+
+    This will create and run the container image.
+
+4. Run the Personfinder local server and test it from host's web browser (https://<container-ip-address>:8000
```

Comment:
Yes, I have not such a configuration, so I can not try it, but I pushed a
new version of run-container.sh script that forwards the PF's ports to the
host. You should be able to connect to PF at the address `localhost:8000`

On Tue, 15 Sep 2015 at 09:45 Hiroshi Ichikawa <notifications@github.com>
wrote:

> In docker/README.md
> <https://github.com/google/personfinder/pull/133#discussion_r39482449>:
>
> > +
> > +    $ docker build -t personfinder-image
> > +
> > +    This will take a while and will create the docker image.
> > +
> > +3. Create the container running
> > +
> > +    $ ./run-container.sh <image-name> [<container-name>]
> > +
> > +    e.g.
> > +
> > +    $ ./run-container.sh personfinder-image personfinder-container  # Container's name is optional
> > +
> > +    This will create and run the container image.
> > +
> > +4. Run the Personfinder local server and test it from host's web browser (https://<container-ip-address>:8000
>
> This doesn't work well with our corporate environment which has HTTP
> proxy. Is there a way to forward the port to a port in the host machine
> instead of accessing container-ip-address?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/personfinder/pull/133/files#r39482449>.
>